tests/output/InteractionsTest.py

```diff
@@ -4,14 +4,13 @@
 
 from coalib.output.Interactions import fail_acquire_settings
 from coalib.output.printers.LogPrinter import LogPrinter
-from coalib.settings.Section import Section
 
 
 class InteractionsTest(unittest.TestCase):
 
     def test_(self):
         log_printer = LogPrinter(NullPrinter())
-        section = Section('')
+        section = None
```

Comment:
I would add another test function `test_section_deprecation` which catches the warning message `fail_acquire_settings: section parameter is deprecated.`
You can use
```
from testfixtures import LogCapture

with LogCapture() as capture:
    capture.check(
            ('root', 'INFO', 'Sample message 1'),
            ('root', 'INFO', 'Sample message 2'),
            ('root', 'INFO', 'Sample message 3')
    )
```